src/explore-education-statistics-common/src/modules/charts/util/__tests__/getMinorAxisSize.test.ts

```diff
@@ -0,0 +1,71 @@
+import getMinorAxisSize from '@common/modules/charts/util/getMinorAxisSize';
+
+describe('getMinorAxisSize', () => {
+  const testDataSetCategories = [
+    {
+      dataSets: {
+        key: {
+          dataSet: { indicator: '', filters: [] },
+          value: 200,
+        },
+      },
+      filter: { value: '', label: '', id: '' },
+    },
+    {
+      dataSets: {
+        key: {
+          dataSet: { indicator: '', filters: [] },
+          value: 20000,
+        },
+      },
+      filter: { value: '', label: '', id: '' },
+    },
+    {
+      dataSets: {
+        key: {
+          dataSet: { indicator: '', filters: [] },
+          value: 6,
+        },
+      },
+      filter: { value: '', label: '', id: '' },
+    },
+  ];
+  test('it returns the size based on the length of the higest value', () => {
```

Comment:
Nitpick - but would add line break to separate this case from the test data

Reply:
Also typo in "higest" :)